test/test_multiply.js

```diff
@@ -0,0 +1,9 @@
+// const multiply = require("../util/multiply");
```

Comment:
I assume you verified these worked and then commented out?

Reply:
Yes I did. Once I got proof of concept I commented them out to clean up the console output